community/kernel/src/main/java/org/neo4j/kernel/api/schema/LabelSchemaDescriptor.java

```diff
@@ -88,7 +88,7 @@ public int getPropertyId()
         if ( propertyIds.length != 1 )
         {
             throw new IllegalStateException(
-                    "Single property schema requires one property but had " + propertyIds.length );
+                    "Single property schema required one property but had " + propertyIds.length );
```

Comment:
The previous version was more correct. The requirement is still in place, so present tense should be used. The new version implies that the schema _used to require_, a requirement that is no longer in force, which is not true.